threads.php

```diff
@@ -135,12 +135,14 @@ function cfth_thread_notice($posts, $query) {
 
 function cfth_timeline_posts($term_id) {
 	$term = get_term_by('id', $term_id, 'threads');
+	$term_meta = get_term_meta( $term_id, 'threads_meta', true );
```

Comment:
This function was introduced as of 4.4.0. Using it here means that the plugin's required minimum WP version will be 4.4.0. That means an update to the README and the plugin's heading docblock.

We should also, for good measure, confirm this function exists using `if ( function_exists( 'get_term_meta' ) )` simply because it is so new